Dockerfile.template

```diff
@@ -50,13 +50,13 @@ RUN set -eux; \
 	} >> /usr/local/etc/gemrc
 
 ENV LANG C.UTF-8
-{{ if .version | . == "3.0.6" or . == "3.1.4" or . == "3.2.2" then ( -}}
```

Comment:
@tianon This made our `RUBY_MAJOR` env var we're relying on in our Dockerfile vanish. We're using this env var for setting the `BUNDLE_BIN_PATH` (e.g. `vendor/bundle/ruby/3.1.0/...`). So we were using ruby 3.1.4 and now with 3.1.5, the BUNDLE_BIN_PATH became `vendor/bundle/ruby/.0`. Sure, I can workaround this, but the change seems wrong and it might affect others!

Reply:
Sorry for the breakage, but it is a variable we never intended to expose (and with a name that isn't really accurate for what it is).

Reply:
didn't know and who else might not know it. we're deploying a fix right now. thanks